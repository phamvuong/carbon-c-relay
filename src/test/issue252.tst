collectd.test_host.cpu.42.percent.idle
