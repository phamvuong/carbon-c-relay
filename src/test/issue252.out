listen
    linemode
        2003 proto tcp
        2003 proto udp
        /tmp/.s.carbon-c-relay.2003 proto unix
    ;

statistics
    submit every 60 seconds
    prefix with carbon.relays.test_hostname
    ;

cluster default
    fnv1a_ch replication 1
        127.0.0.1:2103
    ;

aggregate
        ^collectd\.(.+)\.cpu\.([0-9]+)\.percent\.idle
        ^collectd\.(ABC-.+)\.cpu\.([0-9]+)\.percent\.idle
        ^collectd\.ABC-(.+)\.cpu\.([0-9]+)\.percent\.idle
        ^collectd\.([^.]+)\.cpu\.([0-9]+)\.percent\.idle
    every 10 seconds
    expire after 15 seconds
    timestamp at end of bucket
    compute count write to
        collectd.\1.aggregated.cpu.total_cores
    ;
match *
    send to default
    stop
    ;

aggregation
    ^collectd\.(.+)\.cpu\.([0-9]+)\.percent\.idle (regex) -> collectd.test_host.cpu.42.percent.idle
    count(collectd.\1.aggregated.cpu.total_cores) -> collectd.test_host.aggregated.cpu.total_cores 
aggregation
    ^collectd\.([^.]+)\.cpu\.([0-9]+)\.percent\.idle (regex) -> collectd.test_host.cpu.42.percent.idle
    count(collectd.\1.aggregated.cpu.total_cores) -> collectd.test_host.aggregated.cpu.total_cores 
match
    * -> collectd.test_host.cpu.42.percent.idle
    fnv1a_ch(default)
        127.0.0.1:2103
    stop
